### PR TITLE
new image styles for staff profile squashed

### PR DESCRIPTION
**Pull request by emb03**
commit 38d613971720af6ef6f3408dbaf36b844803b619
Author: Elaine Blakeman <elaineblakeman@library.ucla.edu>
Date:   Wed Jun 8 14:59:40 2016 -0700

    DRUP-915: #ready removes duplicate code from staff_profile_module

commit 301bc10e16f0dd125ae893ebda9ade70790b4224
Author: Elaine Blakeman <elaineblakeman@library.ucla.edu>
Date:   Wed Jun 8 09:22:07 2016 -0700

    DRUP-915: #ready adds two new image styles to staff profile for profile photo and a-z list

### CODE REVIEW

**Comment by emb03** [comment]
@z3cka Please review. I think this should be good to go to test now, unless you want to pull it down.

**Comment by z3cka** [comment]
@emb03 reviewing now _(sorry for the delay)_

File: www/sites/all/modules/features/uclalib_staff_profile/uclalib_staff_profile.info [hunk 1]
```diff
@@ -2,7 +2,7 @@ name = UCLALIB Staff Profile
 description = Staff Profile content type
 core = 7.x
 package = UCLALIB
-version = 7.x-1.3
+version = 7.x-1.3-beta1
```
Comment (z3cka) [inline www/sites/all/modules/features/uclalib_staff_profile/uclalib_staff_profile.info]: this should be `version = 7.x-1.4-beta1`

**Comment by z3cka** [comment]
let's bump to version 1.4-beta1 and put "DRUP-915: #ready ..." in the title of the PR and we should be good to go.

**Pull request closed without merging**